apps/server/src/schema/feedback/feedback-operation.ts

```diff
@@ -0,0 +1,36 @@
+import { prisma } from '@repo/database';
+import { FireAndForget, messageSchema } from '@repo/utils';
+import { builder } from '../builder';
+import { sendFeedbackReceivedEmail } from '../../email';
+import { FeedbackDto, FeedbackType } from './feedback-types';
+
+const fireAndForget = new FireAndForget();
+
+builder.mutationFields((t) => ({
+  sendFeedback: t.withAuth({ authenticated: true }).prismaField({
+    nullable: false,
+    type: FeedbackDto,
+    args: {
+      message: t.arg.string({ required: true, validate: { schema: messageSchema } }),
+      type: t.arg({
+        type: FeedbackType,
+        required: true,
+      }),
+    },
+    resolve: async (query, _root, args, ctx, _info) => {
+      const { message, type } = args;
+
+      const user = await prisma.user.findUniqueOrThrow({ where: { id: ctx.currentUserId } });
+
+      fireAndForget.add(() => sendFeedbackReceivedEmail(user.email, user.firstName, user.lastName));
+
+      return await prisma.feedback.create({
```

Comment:
```suggestion
      return await prisma.feedback.create({
              ...query,
```
You are right, I didn't notice that we haven't exposed everything in FeedbackDto. I added the relevant code.

you are not using the query variable from the resolve. That is code smell, we always need to use this to enable graphql joins. For example this:
```
mutation sendFeedback {
  sendFeedback(message: "da", type: BUG_REPORT) {
    user {
      id
    }
  }
}
```
won't be able to work without the `...query`